io.github.complexlogic.rsgain.json

```diff
@@ -0,0 +1,74 @@
+{
+    "app-id": "io.github.complexlogic.rsgain",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "rsgain",
+    "finish-args": [
+        "--filesystem=home",
+        "--filesystem=xdg-music"
```

Comment:
no display permissions?

Reply:
The program has a command line interface, so no display permissions are needed.

Reply:
then why does it pull the KDE runtime?

Reply:
also cli application are not good candidates for flatpak.

Reply:
The reason for the KDE runtime is because the app depends on TagLib which is not available in the freedesktop runtime.

I could also build TagLib as a separate module.

Reply:
Is it a rule that no CLI applications are allowed in flathub?

Reply:
I just updated it so now TagLib is built within the manifest, so it no longer depends on the KDE runtime.

Reply:
is it supposed to playback music?